src/Contract/Transaction.purs

```diff
@@ -274,6 +269,9 @@ import Effect.Class (liftEffect)
 import Effect.Exception (throw)
 import Untagged.Union (asOneOf)
 
+-- | NOTE: `scriptOutputToTransactionOutput` is deprecated. See
```

Comment:
What this comment is for? Please remove

Reply:
sorry, I think I misunderstood Josephs instructions. removed a similar NOTE from `Contract.Address` as well.